parent/pom.xml

```diff
@@ -109,19 +109,27 @@
     <groovy-eclipse-compiler.version>2.9.3-01</groovy-eclipse-compiler.version>
     <groovy-eclipse-batch.version>2.4.16-03</groovy-eclipse-batch.version>
     <client-runtime.version.v2>2.1.1</client-runtime.version.v2>
-    <slf4j-api.version>1.7.5</slf4j-api.version>
+    <slf4j-api.version>1.7.6</slf4j-api.version>
     <slf4j-log4j12.version>1.7.0</slf4j-log4j12.version>
     <reactor-netty.version>0.8.3.RELEASE</reactor-netty.version>
     <netty.version>4.1.33.Final</netty.version>
     <async-http-client.version>2.5.2</async-http-client.version>
     <opencensus.version>0.20.0</opencensus.version>
     <rxjava.version>1.2.4</rxjava.version>
-    <guava.version>24.1.1-jre</guava.version>
+    <guava.version>27.0.1-jre</guava.version>
```

Comment:
Note these version changes also require you to update the version of this pom file.

Reply:
updated versions of parent/pom.xml, pom.data.xml, pom.client.xml

Reply:
Reverted the parent pom to guava 24.1.1
Updated Cosmos parent pom to use 27.0.1